pages.de/linux/apt-add-repository.md

```diff
@@ -2,18 +2,18 @@
 
 > Editiere die Repository-Listen.
 
-- Fügt ein neues Repository hinzu:
+- Füge ein neues Repository hinzu:
 
-`apt-add-repository {{repository_spec}}`
+`apt-add-repository {{repository}}`
```

Comment:
@bl-ue what do you think about this?

Reply:
I'd say keep `repository`, because `repository_spec` may not be obvious to new users (like me.)  I have not much of an opinion, so if you feel strongly about it, we can change it. 👍🏻

Reply:
Agreed. I'd suggest keeping `repository` too.